sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -153,6 +174,35 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putString(KEY_TRACKING_OPTIONS, optionsJson) }
     }
 
+    private fun getFallbackTrackingOptions(context: Context): RadarTrackingOptions? {
```

Comment:
Also food for thought (this could be a later sdk refactor) it may make sense to start thinking about splitting up the RadarSettings object into more feature/single-responsibility classes, so there could be a TrackingOptionsSettings, ForegroundTrackingSettings, etc. Might also help reduce complexity and class sizes. Implementation would be the same/each could extend from a base settings class.

Reply:
yeah, this is starting to get unwieldy. I think we could combine this with the key namespacing refactor you proposed above in a later release.